appeals/web/src/styles/7-components/show-more.scss

```diff
@@ -13,10 +13,8 @@
 		text-decoration: underline;
 	}
 
-	&__content {
-		mark {
-			background-color: #f4cdc6
-		}
+	mark {
```

Comment:
Is this overriding a default GDS style? Just wondering if removing this entirely would also fix the issue in a more global way